### PR TITLE
Support for Loki Datasource

### LINKED ISSUE #92: Support for Loki Datasource
##### SUMMARY
Support for Loki Datasources in grafana_datasource module.

##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME
grafana_datasource

##### ADDITIONAL INFORMATION
Currently it's not possible to add a loki datasource with the grafana_datasource module in this collection.

It would be nice to add one with a task like this:
```yaml
- name: Ensure loki datasource exists.
  grafana_datasource:
    name: "datasource-loki"
    grafana_url: "https://grafana.company.com"
    grafana_user: "admin"
    grafana_password: "xxxxxx"
    org_id: "1"
    ds_type: "loki"
    ds_url: "https://loki.company.com:3100"
```

I took a brief look at the [grafana_datasource](https://github.com/ansible-collections/community.grafana/blob/master/plugins/modules/grafana_datasource.py) module and it seems at first glance that it just has to be added here: https://github.com/ansible-collections/community.grafana/blob/5ecea6c4ffd91dde5b7ce78478a646c3880817e8/plugins/modules/grafana_datasource.py#L556-L565
If so, I can open a PR for this.

### PR DESCRIPTION
**Pull request by Robinhuett**
##### SUMMARY

This PR adds support for Loki datasources including tests.

Fixes #92 

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

grafana_datasource

##### ADDITIONAL INFORMATION

While I was testing I noticed that not all test used the variables defined in the defaults.
I added this in a separate commit, but if you like I can also open a second PR for this.

### CODE REVIEW

File: tests/integration/targets/grafana_datasource/tasks/loki.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+- debug:
+    var: result
+
+- assert:
+    that:
+    - result.changed
+    - "result.msg == 'Datasource datasource-loki deleted.'"
+
+- name: Delete cloudwatch datasource (idempotency)
```
Comment (rrey) [inline tests/integration/targets/grafana_datasource/tasks/loki.yml:71]: s/cloudwatch/loki/

Comment (Robinhuett) [reply tests/integration/targets/grafana_datasource/tasks/loki.yml:71]: Ohh thanks, I missed that one. I'll update it in a sec.

**Comment by rrey** [comment]
Thx for contributing @robinhuett !

**Comment by Robinhuett** [comment]
It's a pleasure.

**Pull request merged**